packages/story-editor/src/output/test/page.js

```diff
@@ -1243,6 +1243,41 @@ describe('Page output', () => {
         'background-audio="https://example.com/audio.mp3"'
       );
     });
+    it('should add background audio as amp-video', async () => {
+      const props = {
+        id: '123',
+        page: {
+          backgroundAudio: {
+            src: 'https://example.com/audio.mp3',
+            id: 123,
+            mimeType: 'audio/mpeg',
+          },
+          tracks: [
+            {
+              track: 'https://example.com/track.vtt',
+              trackId: 123,
+              trackName: 'track.vtt',
+              id: 'rersd-fdfd-fdfd-fdfd',
+              srcLang: '',
+              label: '',
+              kind: 'captions',
+            },
+          ],
+          id: '123',
+          elements: [],
+        },
+        autoAdvance: false,
+        defaultPageDuration: 7,
+      };
+
+      const { container } = render(<PageOutput {...props} />);
```

Comment:
We'd want an assertion here that the `background-audio` attribute on `amp-story-page` is not set if we have captions.

Reply:
Could also add an assertion that the ID of `amp-story-captions` matches the video ID.

The snapshot test doesn't make that obvious.

Aside: why the snapshot test? Seems to be the first one in this file. Just curious.

Reply:
> Aside: why the snapshot test? Seems to be the first one in this file. Just curious.

I wanted to avoid generating a snapshot file. But as amp video tag was complex, it could not be avoided. I thought, if we already have a snapshot file for this test, why not use it in both spaces. 

> Could also add an assertion that the ID of amp-story-captions matches the video ID. 

Do we do this for other amp-video / captions? If not, why is it needed here. 

> We'd want an assertion here that the background-audio attribute on amp-story-page is not set if we have captions.

Good idea!

Reply:
Added a test

Reply:
> Do we do this for other amp-video / captions? If not, why is it needed here.

I don't know, but we should add one.

Because right now it's incorrect as per my other message https://github.com/GoogleForCreators/web-stories-wp/pull/10288#discussion_r790741798

So a test would catch this bug.

Reply:
Fixed.